.github/workflows/python-validate.yml

```diff
@@ -50,13 +52,6 @@ jobs:
           cache-dependency-path: |
             **/pyproject.toml
 
-      - name: Install SDK project and dependencies
```

Comment:
Why this is removed? (even if applicable, doesn't feel as scope of this PR)

Reply:
I've removed it to make sure base sdk is installed from pypi registry, instead of relying on local installation. Does that make sense?

Reply:
But I believe this installation is needed to validate and test SDK project itself (note we're in context of validation script that tests all packages not just `aws-lambda-sdk`)

Reply:
For readability of CI output, it's better if _installation_ step is distinguished from one where we run unit tests.

e.g. in Node.js validate we have separate installation steps for each package and then separate for each unit tests run in each package.

Having that when something goes wrong it's easier to find out where it stuck

Reply:
I see, makes sense 👍